applications/asset_tracker_v2/src/cloud/nrf_cloud_integration.c

```diff
@@ -67,14 +116,15 @@ static void user_association_work_fn(struct k_work *work)
 static int send_service_info(void)
 {
 	int err;
-	bool fota_capable = IS_ENABLED(CONFIG_NRF_CLOUD_FOTA) &&
-			    IS_ENABLED(CONFIG_BOOTLOADER_MCUBOOT);
-	bool boot_fota_capable = IS_ENABLED(CONFIG_BUILD_S1_VARIANT) &&
-				 IS_ENABLED(CONFIG_SECURE_BOOT);
 	struct nrf_cloud_svc_info_fota fota_info = {
-		.application = fota_capable,
-		.bootloader = fota_capable && boot_fota_capable,
-		.modem = fota_capable
+#if defined(CONFIG_NRF_CLOUD_FOTA)
```

Comment:
Beyond the scope of this PR, but I think this is an argument for `nrf_cloud_fota_is_type_enabled` to be available even when `CONFIG_NRF_CLOUD_FOTA` is disabled. If it's disabled, they should simply return false

Reply:
sure, it can be moved into fota common.
it was in nrf_cloud_fota just so it wouldn't incorrectly used for REST FOTA.

the #if guard is probably overkill here anyway since FOTA is enabled by default.
i don't most think most user applications are going to be developed to handle all varieties of configurations.

Reply:
added a commit as discussed:
https://github.com/nrfconnect/sdk-nrf/pull/8407/commits/d4549e71ec05592e8df2b64f1220f7f3e58ced09